identifiedLight[] lights
sensor_msgs/Image img
float64 timestamp
